### PR TITLE
Add query to check if task definitions contains passwords. Closes #2321

### LINKED ISSUE #2321: Add "ECS Task Definition Container With Plaintext Password" query for CloudFormation
### Platform
CloudFormation

### Description
Add support in CloudFormation for the query already implemented in Terraform and Ansible.
It's not recommended to use plaintext environment variables for sensitive information, such as credential data.

### PR DESCRIPTION
**Pull request by joaomartinscx**
Closes #2321

**Proposed Changes**
Besides adding the query in CloudFormation, also changed the query names in Terraform and ansible to match the unified name.

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/yaml.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/yaml.json:1]: Remove this file?

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/jason.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/jason.json:1]: Remove this file?

File: assets/queries/ansible/aws/ecs_task_definition_container_has_password/metadata.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
```
Comment (Ruben-Silva) [inline assets/queries/ansible/aws/ecs_task_definition_container_has_password/metadata.json:1]: Didn't change the name of the folder? Same applies in Terraform.

**Review by Ruben-Silva — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/yaml.json (hunk 1 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/yaml.json]: Done

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/jason.json (hunk 2 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/jason.json]: Done

File: assets/queries/ansible/aws/ecs_task_definition_container_has_password/metadata.json (hunk 3 above)
Comment (joaomartinscx) [reply assets/queries/ansible/aws/ecs_task_definition_container_has_password/metadata.json:1]: Done

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/negative.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  taskdefinition:
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/negative.yaml:4]: Change "taskdefinition" to "taskdefinition2". Sorry I missed this earlier.

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "A sample template",
+  "Resources": {
+    "taskdefinition": {
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.json:5]: Change "taskdefinition" to "taskdefinition3". Sorry I missed this earlier.

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  taskdefinition:
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.yaml:4]: Change "taskdefinition" to "taskdefinition4". Sorry I missed this earlier.

**Review by Ruben-Silva — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/negative.yaml (hunk 4 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/negative.yaml]: done

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.json (hunk 5 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.json]: done

File: assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.yaml (hunk 6 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.yaml]: done

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**